docs/docusaurus.config.js

```diff
@@ -24,8 +24,8 @@ const config = {
   deploymentBranch: "gh-pages", // Branch that GitHub pages will deploy from.
   trailingSlash: false,
 
-  onBrokenLinks: "throw",
-  onBrokenMarkdownLinks: "warn",
+  onBrokenLinks: "ignore",
+  onBrokenMarkdownLinks: "ignore",
```

Comment:
These tests are very valuable for catching docs mistakes.  Without this test, we'll get lots of docs bugs.  I'm surprised `onBrokenMarkdownLinks` wasn't configured for failure before - we should actually turn that back on.